### ISSUE TITLE
Kafka instance creation is always failing

### ISSUE THREAD

**Issue by franvila** [bf2fc6cc711aee1a0c2a/kas-installer#141]
Since yesterday, when executing

`./managed_kafka.sh --create instance`

We always receive the status failed indefinitely.
![Screenshot from 2022-02-16 11-51-07](https://user-images.githubusercontent.com/57452611/154250181-a22a9776-7c27-44ad-9808-7d422db36520.png)

After running `./managed_kafka.sh --list`, the output is:
```
{
  "kind": "KafkaRequestList",
  "page": 1,
  "size": 1,
  "total": 1,
  "items": [
    {
      "id": "c86dbc2in864m3gvmfl0",
      "kind": "Kafka",
      "href": "/api/kafkas_mgmt/v1/kafkas/c86dbc2in864m3gvmfl0",
      "status": "failed",
      "cloud_provider": "aws",
      "multi_az": true,
      "region": "us-east-1",
      "owner": "fvila_kafka_sre",
      "name": "instance",
      "created_at": "2022-02-16T10:45:04.398942Z",
      "updated_at": "2022-02-16T10:50:31.890442Z",
      "failed_reason": "failed to get desired Strimzi version c86dbc2in864m3gvmfl0",
      "instance_type": "eval",
      "reauthentication_enabled": true,
      "kafka_storage_size": "1000Gi"
    }
  ]
}
```

**Comment by MikeEdgar** [comment]
@franvila - there is an issue in newer fleet-manager versions. Try setting these to set the last known working tag:

```
KAS_FLEET_MANAGER_BF2_REF=53e2bc49b77cad4a61398e557b17dfb3d6e483b5
KAS_FLEET_MANAGER_IMAGE_TAG=53e2bc4
```

**Comment by agullon** [comment]
Github commit id is stale, please use this for now on:
`KAS_FLEET_MANAGER_BF2_REF=593e875d1dcb7d79bd4f85789c55c5341aea2db9
`

**Comment by k-wall** [comment]
@pb82 FYI 
It appears there was some change on the fleetmanager a few weeks back that broke kas-installer.  I'll see if I can get together sufficient details to raise an issue.

**Comment by pb82** [comment]
@k-wall thanks. What was the last image that was working? We can create a diff from that.

**Comment by rareddy** [comment]
were u able to create the cluster @k-wall with above commit REF? I am going to give it try now.

**Comment by MikeEdgar** [comment]
@pb82 the last working commit was `593e875d1dcb7d79bd4f85789c55c5341aea2db9`, which corresponds to image ref `53e2bc4`

**Comment by MikeEdgar** [comment]
Fixed by #137
